addons/website/static/src/components/dialog/add_page_dialog.xml

```diff
@@ -36,7 +36,7 @@
 
 <t t-name="website.AddPageTemplatePreview">
     <t t-call="website.AddPageTemplatePreviewWrapper">
-        <iframe class="border-0 pe-none transition-base" t-ref="iframe" src="about:blank"/>
+        <iframe class="border-0 pe-none transition-base" t-ref="iframe" src="about:blank" title="Add page template"/>
```

Comment:
Same comment as the previous one: I am really curious about this one. Also won't that mean a title will appear as a popup everywhere?

Reply:
You are right, I have rather added `aria-label` for `iframe` :+1: